src/dtx/dtx_srv.c

```diff
@@ -309,6 +309,24 @@ dtx_init(void)
 		dtx_yield_cycle = DTX_YIELD_DEF;
 	}
 
+	str = getenv("DTX_AGG_THRESHOLD");
```

Comment:
Since this controls the count threshold, but not the age threshold, should this variable be more specific in case we want to make the age threshold configurable at some point?
E.g. `DTX_AGG_THRESHOLD_CNT, DTX_AGG_THRESHOLD_COUNT`
Then we leave room for this later: `DTX_AGG_THRESHOLD_AGE`

Reply:
Good point. I will add more control for the DTX aggregation age. But please note that, as long as one threshold (total count or oldest age) is exceeded, DTX aggregation will be triggered.

Reply:
Thanks! For my uses, I think it will be helpful to use either the count or the age, depending on the test ran